gateway/unsubscribe_transaction.go

```diff
@@ -0,0 +1,48 @@
+// MQTT-SN SUBACK contains TopicID field. We must use this transaction to save
```

Comment:
This is copy&paste error: `UNSUBACK` does not contain `TopicID`.

Reply:
This comment is no longer relevant. The file in question has been deleted.